members/P000595.yaml

```diff
@@ -1,62 +1,211 @@
 bioguide: P000595
 contact_form:
   method: post
-  action: /content/contact-gary
+  action: "https://www.peters.senate.gov/contact/email-gary"
   steps:
-    - visit: "http://www.peters.senate.gov/content/contact-gary"
+    - visit: "https://www.peters.senate.gov/contact/email-gary"
+    - select:
+        - name: actions
+          selector: "#actions"
+          value: comment
+          required: true
+          options:
+            Comment on an Issue: opinion
+            Help with a Federal Agency: Help with a Federal Agency
+            Meeting Requests: Meeting Requests
+            Tour Requests: Tour Requests
+            Office Locations: Office Locations
+            Student Resources: Student Resources
+            Veterans Resources: Veterans Resources
+    - wait:
+        - value: 10
```

Comment:
`wait` set at 1 second works. When using `find`, only the `select` form elements have values entered, and the `fill_in` elements are left blank.